main.tf

```diff
@@ -158,6 +158,7 @@ data "template_file" "runners" {
     runners_root_size                 = "${var.runners_root_size}"
     runners_iam_instance_profile_name = "${var.runners_iam_instance_profile_name}"
     runners_use_private_address       = "${var.runners_use_private_address}"
+    runners_environment               = "${jsonencode(var.runners_environment)}"
```

Comment:
May suggest to rename the var to `runners_environment_vars` slight more clear.

Reply:
Good idea, done.